test/adapters/postgres.test.js

```diff
@@ -38,46 +37,43 @@ if (process.env.PATIO_DB === "pg") {
                 this.sqls.push(sql.trim());
                 return origExecute.apply(this, arguments);
             };
-            return comb.serial([
```

Comment:
I believe we deprecated comb.serial because using promises returned from patio functions was easier to understand.